app/components/Inspector/Screenshot.js

```diff
@@ -46,17 +51,25 @@ export default class Screenshot extends Component {
     // Recurse through the 'source' JSON and render a highlighter rect for each element
     const highlighterRects = [];
 
+    let highlighterXOffset = 0;
+    if (this.containerEl) {
+      const screenshotEl = this.containerEl.querySelector('img');
+      highlighterXOffset = screenshotEl.getBoundingClientRect().left -
+                           this.containerEl.getBoundingClientRect().left;
+    }
+
```

Comment:
No, this change is because I wanted to ensure that the image was always horizontally centered in the bounding box of the overall column. Given that, the top left of the image is no longer guaranteed to be at `0, 0` in the bounding column. This we may need to offset the X position based on where the top left of the image is in the column.